sulis_module/src/inventory_builder.rs

```diff
@@ -142,7 +142,7 @@ impl InventoryBuilder {
                     None
                 }
                 Some(item) => {
-                    let state = ItemState::new(item, entry.item.variant);
+                    let state = ItemState::new(item, entry.item.variant, true);
```

Comment:
why isn't unidentified configurable in this context?  (i.e. why is it always true)

Reply:
I thought this generated player character items at the start, do we ever want to add a pc that has a cursed item from start? Oh well, it could be an interesting start perspective, so added.